src/ansys/simai/core/data/geomai/predictions.py

```diff
@@ -49,11 +49,16 @@ class GeomAIPredictionConfiguration(BaseModel):
     Required.
     """
     resolution: Optional[Tuple[PositiveInt, PositiveInt, PositiveInt]] = None
-    """A list of three integers defining the number of divisions along the X, Y, and Z axes.
+    """A list of three integers defining the number of voxels along the X, Y, and Z axes.
 
-    Higher values allow for more detailed geometry reconstruction, while lower values reduce detail but speed up computation.
+    Use higher resolution for complex or precise geometries, and lower resolution for simple shapes or quick previews.
 
-    Defaults to ``[100, 100, 100]`` if none is provided.
+    The total number of voxels must not exceed 900^3, that is the three numbers multiplied together must be less than or equal to 900^3.
```

Comment:
```suggestion
    The total number of voxels must not exceed 900^3, that is the three dimensions multiplied together must be less than or equal to 900^3.
```
or `x, y, z multiplied...` ?
That's just a comment, if you think it's not better than the actual version keep it that way 👍

Reply:
`x, y, z multiplied...` could be more simple to understand yes!